CMakeLists.txt

```diff
@@ -86,6 +89,9 @@ daq_add_unit_test(NamedObject_test		  )
 
 daq_install(TARGETS daq_application 
                     appfwk
-		    appfwk_QueryResponseCommandFacility_duneCommandFacility 
-                    VectorIntFanOutDAQModule_duneDAQModule StringFanOutDAQModule_duneDAQModule NonCopyableTypeFanOutDAQModule_duneDAQModule
+		    # appfwk_QueryResponseCommandFacility_duneCommandFacility 
```

Comment:
Uh?

Reply:
Are you asking about the commented-out QRCF, @mroda88?

In my git history I see this coming in through the start of @alessandrothea's branch but maybe that got picked up from where he started which may have been my branch.  In any case, let me give my view:

The idea of a QRCF is definitely not gone.  But since the whole notion of command handling changed, a QRCF needs a revamp.  There are a few ways to provide it and it's on my list to try some out and see what might work well.  Some approaches:

0) the file command facility (FCF) can take a sequence of commands, each with an optional time delay to "play" the sequence at some pace.  This is not at all interactive like QRCF, but can be used now.

1) I think a user can (almost?) use the FCF now with individual command files via `cat cmd.json > fifo.jsonstream` to deliver one command, where the file here is made via "mkfifo".  Then, the user can have a family of `.json` files ready and use the shell command line as sort of a cheap REPL.  I say "almost" because I need to test this to see how the FCF handles `cat` closing the fifo on each command.  Probably some C++ needs improvement to support this.

2) Essentially the same but instead of a fifo + the FCF the command would be delivered through @roland-sipos embedded HTTP server CF.

3) Provide support code in Python for defining the family of possible commands, use eg `ipython` or `ptipython` to provide bespoked REPL.

4) Of course, a REPL could be implemented in C++ like the previous QRCF.

Reply:
Yes, `QueryResponseCommandFacility` was commented out simply because in its current form can only provide the command and not the command-data payload.

Reply:
FYI: The command facility is scheduled to be moved in a different package as a standalone generic tool.